_episodes/04-recipe.md

```diff
@@ -330,331 +73,545 @@ Please note the following sections:
+>
+> > ## Answers
+> >
+> >   1. The example recipe is written by Bouwe Andela and Mattia Righi.
+> >   1. Manual Schlund is listed as the maintainer of this recipe.
+> >   1. Two datasets are analysed:
+> >      - CMIP6 data from the model BCC-ESM1
+> >      - CMIP5 data from the model CANESM2
+> >   1. The preprocessor `annual_mean_global` computes an area mean as well as
+> >      annual means
+> >   1. The diagnostic called `map` executes a script referred to as 'script1'.
+> >      This is a python script named `examples/diagnostic.py`
+> >   1. There are two diagnostics: `map` and `timeseries`. Under the diagnostic
+> >      `map` we find two tasks:
+> >      - a preprocessor task called `tas`, applying the preprocessor called
+> >        `select_january` to the variable `tas`.
+> >      - a diagnostic task called `script1`, applying the script
+> >        `examples/diagnostic.py` to the preprocessed data (`map/tas`).
+> >
+> >      Under the diagnostic `timeseries` we find three tasks:
+> >      - a preprocessor task called `tas_amsterdam`, applying the preprocessor
+> >        called `annual_mean_amsterdam` to the variable `tas`.
+> >      - a preprocessor task called `tas_global`, applying the preprocessor
+> >        called `annual_mean_global` to the variable `tas`.
+> >      - a diagnostic task called `script1`, applying the script
+> >        `examples/diagnostic.py` to the preprocessed data
+> >        (timeseries/tas_global and timeseries/tas_amsterdam).
+> >
+> {: .solution}
+{: .challenge}
 
-> ##  Your settings.yml file
->
-> This file is located at `./recipe_example_#_#/run/diag_timeseries_temperature/timeseries_diag/settings.yaml`.
-> Let's have look at the file:
+> ## Pro tip: short names and variable groups
 >
-> ~~~bash
-> nano ./recipe_example_#_#/run/diag_timeseries_temperature/timeseries_diag/settings.yaml
-> ~~~
->
-{: .solution}
+> The preprocessor tasks in ESMValTool are called 'variable groups'. For the
+> diagnostic `timeseries`, we have two variable groups: `tas_amsterdam` and
+> `tas_global`. Both of them operate on the variable `tas` (as indicated by the
+> `short_name`), but they apply different preprocessors. For the diagnostic
+> `map` the variable group itself is named `tas`, and you'll notice that we're
+> not explicitly providing the `short_name`. This is a shorthand build into
+> ESMValTool.
+{: .callout}
 
-> ## A metadata.yml file.
->
-> This file is located at `./recipe_example_#_#/preproc/diag_timeseries_temperature/timeseries_variable/metadata.yml`.
-> Please note that `preproc` diretctory exist if only `remove_preproc_dir: false`
-> in ``config-user.yml`` file.
-> Let's have look at the file:
->
-> ~~~bash
-> nano ./recipe_example_#_#/preproc/diag_timeseries_temperature/timeseries_variable/metadata.yml
-> ~~~
->
-{: .solution}
+> ## Output files
+>
+> Have another look at the output directory created by the ESMValTool run.
```

Comment:
Should this point to the work directory? Because my output directory does not have the directories listed below.

Reply:
I think in the config episode we are setting the output_dir, right? 
Otherwise, we can use the log output of ESMValTool to locate it.